src/sql-lexer/src/keywords.txt

```diff
@@ -108,6 +108,7 @@ Decimal
 Declare
 Decorrelated
 Default
+Defaults
```

Comment:
Is it okay to add this new keyword?

Reply:
yeah, it shouldn't make parsing ambiguous

Reply:
ty